statistics/selectivity_test.go

```diff
@@ -174,3 +190,34 @@ func (s *testSelectivitySuite) TestSelectivity(c *C) {
 		c.Assert(math.Abs(ratio-tt.selectivity) < eps, IsTrue, Commentf("for %s, needed: %v, got: %v", tt.exprs, tt.selectivity, ratio))
 	}
 }
+
+func BenchmarkSelectivity(b *testing.B) {
+	c := &C{}
+	s := &testSelectivitySuite{}
```

Comment:
It's a benchmark, we don't need to use the gocheck framework.

Reply:
To avoid repeat code here.

Reply:
It's just a struct, if you don't view is as a framework.